extensions/devservices/common/src/main/java/io/quarkus/devservices/common/ContainerAddress.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkus.devservices.common;
+
+public class ContainerAddress {
+    private final String ipAddress;
+    private final int port;
+
+    public ContainerAddress(String ipAddress, Integer port) {
+        this.ipAddress = ipAddress;
+        this.port = port;
+    }
+
+    public String getIpAddress() {
+        return ipAddress;
+    }
+
+    public int getPort() {
+        return port;
+    }
+
+    public String getUrl() {
+        return String.format("%s:%d", ipAddress, port);
```

Comment:
Should we add an optional scheme? Would that make sense?

Reply:
At the moment it wasn't (isn't) needed - I wanted to keep it simple.